File: src/main/scala/com/github/tetrisanalyzer/Timer.scala
package com.github.tetrisanalyzer

object Timer {
  var timer1 = new Timer("timer1")

  def printAllResult() {
    timer1.printResult
  }
}

class Timer(message: String) {
  private var startMillis: Long = 0L
  private var passed: Long = 0

  def start: Unit = {
    startMillis = System.currentTimeMillis
  }

  def stop: Unit = {
    var stop: Long = System.currentTimeMillis
    passed += (stop - startMillis)
    startMillis = stop
  }

  def printResult: Unit = {
    System.out.println(message + ": " + (passed / 1000.0) + " sec")
  }
}

File: otherlang/src/main/java/com/github/tetrisanalyzer/Timer.java
package com.github.tetrisanalyzer;

public class Timer {
    public static Timer timer1 = new Timer("timer1");
    public static Timer timer2 = new Timer("timer2");

    private long start;
    private String message;
    private long passed = 0;

    public static void printAllResult() {
        timer1.printResult();
    }

    public Timer(String message) {
        this.message = message;
    }

    public void start() {
        start = System.currentTimeMillis();
    }

    public void stop() {
        long stop = System.currentTimeMillis();
        passed += (stop - start);
        start = stop;
    }

    public void printResult() {
        System.out.println(message + ": " + (passed / 1000.0) + " sec");
    }
}
